ee/desktop/runner/runner.go

```diff
@@ -179,6 +181,9 @@ func New(opts ...desktopUsersProcessesRunnerOption) (*DesktopUsersProcessesRunne
 	runner.writeDefaultMenuTemplateFile()
 	runner.refreshMenu()
 
+	// Observe DesktopEnabled changes to know when to enable/disable process spawning
+	runner.knapsack.Flags.RegisterChangeObserver(runner, flags.DesktopEnabled)
```

Comment:
Hrm. This feels clean, but I kinda wish we didn't have to pull in `flags` for it.

Reply:
I moved the `FlagKeys` to a `keys` package for a couple reasons, but it seems more readable too.